pyV2DL3/eventdisplay/fillRESPONSE.py

```diff
@@ -37,15 +38,50 @@ def print_logging_info(irf_to_store, camera_offsets, pedvar, zenith):
 
 
 def check_parameter_range(par, par_irf, par_name):
```

Comment:
can we find a better name for the variable `par_irf`? It is very hard to see what it actually describe.

Or add a description of the arguments to the docstring? I have a hard time to see what is what.